pkg/data/management/role_data.go

```diff
@@ -332,7 +328,6 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 		addRule().apiGroups("management.cattle.io").resources("clustercatalogs").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("projectcatalogs").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("projectmonitorgraphs").verbs("get", "list", "watch").
-		addRule().apiGroups("monitoring.coreos.com").resources("prometheuses", "prometheusrules", "servicemonitors").verbs("get", "list", "watch").
```

Comment:
I think that if a role is tied to `monitoring.cattle.io`, it's safe to remove.

If it is tied to `monitoring.coreos.com`, that could impact Monitoring V2 in unforeseen ways, so this should be kept around IMO.

Also, RBAC related to Monitoring V1 resources (i.e. Notifiers, ProjectMonitorGraphs, ProjectAlertRules, ProjectAlertGroups, ClusterAlertRules, ClusterAlertGroups) should all be removed.

Reply:
@aiyengar2 Added back the roles that have `monitoring.coreos.com` as apiGroup
I also removed the rbac related to `ProjectMonitorGraphs` and removed the other ones in the [alertingV1 PR](https://github.com/rancher/rancher/pull/44211/commits/dbe6d9be3d735f6a85bb7beb1c6f23fce464aaf3)